projects/Mallard/src/helpers/storage.ts

```diff
@@ -96,8 +96,8 @@ const notificationsEnabledCache = createAsyncCache<boolean>(
 	'notificationsEnabled',
 );
 
-const newApiUrlSetForBetaUsers = createAsyncCache<boolean>(
-	'newApiUrlSetForBeta',
+const rollbackedApiUrlForBetaUsers = createAsyncCache<boolean>(
```

Comment:
Same again here. Do we need `rollbacked` in the naming for this?

Reply:
@abeddow91 explained above, I also added a comment on what this rollback means. Eventually, we can delete this whole thing before release this to prod